### DISCUSSION TITLE
How to reset settings?

### DISCUSSION THREAD

**Discussion by COWCATGames** [mgth/LittleBigMouse#280] [Q&A]
I often can't open the control panel. When launching it, this either takes ages or won't open at all. LBP is very buggy for me.

I'd like to reset my configuration but I can't find anywhere this info: where does LBP store its settings? Files or registry? Which files?

**Comment by 4nqyi** [comment]
https://github.com/mgth/LittleBigMouse/issues/147

Delete the registry setting and start fresh.
HKEY_CURRENT_USER\SOFTWARE\Mgth\LittleBigMouse
